pylint/utils.py

```diff
@@ -1145,3 +1145,13 @@ def _rest_format_section(stream, section, options, encoding=None, doc=None):
             value = _encode(_format_option_value(optdict, value), encoding)
             print(file=stream)
             print('  Default: ``%s``' % value.replace("`` ", "```` ``"), file=stream)
+
+
+def get_duplicated(items):
```

Comment:
This can be local in the file where the check is, since it is not yet used by more than one checker. Also, it could be renamed to duplicated(items).